db/transformations.ts

```diff
@@ -0,0 +1,192 @@
+/* eslint-disable unicorn/no-abusive-eslint-disable */
+/* eslint-disable eslint-comments/no-unlimited-disable */
+
+/* eslint-disable canonical/sort-keys */
+
+import { getArgsAsObjectUsingBase64Fallback, getNearAmountConsideringStaking } from '../helpers/converters';
+import { type AccountId, getCurrencyByContractFromNear } from '../helpers/currency';
+import { formatDateFromNano } from '../helpers/datetime';
+
+import { type CsvRow, type IndexerRow } from './Row';
+
+const SYSTEM_ACCOUNT_ID = 'system';
+const BULKSENDER_ACCOUNT_ID = 'bulksender.near';
+const MINIMUM_AMOUNT = 0.000_001;
+const MINIMUM_AMOUNT_FOR_SYSTEM_ACCOUNT = 0.5;
+const YOCTO_CONVERSION_CONSTANT = 10 ** 24;
+
+function getRow(indexerRow: IndexerRow, accountId: AccountId, nearAmount: number, ftAmountIn: string, ftCurrencyIn: string, ftAmountOut = '', ftCurrencyOut = ''): CsvRow {
+  return {
+    date: formatDateFromNano(indexerRow.block_timestamp),
+    account_id: accountId,
+    method_name: String(indexerRow.action_kind === 'TRANSFER' ? 'transfer' : indexerRow.args?.method_name),
+    block_timestamp: indexerRow.block_timestamp,
+    from_account: indexerRow.receipt_predecessor_account_id,
+    block_height: indexerRow.block_height,
+    args: JSON.stringify(getArgsAsObjectUsingBase64Fallback(indexerRow.args)),
+    transaction_hash: indexerRow.transaction_hash,
+    amount_transferred: nearAmount,
+    // Fungible Token
+    ft_amount_out: ftAmountOut,
+    ft_currency_out: ftCurrencyOut,
+    ft_amount_in: ftAmountIn,
+    ft_currency_in: ftCurrencyIn,
+    to_account: indexerRow.receipt_receiver_account_id,
+    amount_staked: getNearAmountConsideringStaking(indexerRow, nearAmount),
+  };
+}
+
+function divide(rawAmount: number, decimals: number): string {
+  // TODO: Document what is happening and why (and improve the function name).
+  return String(rawAmount / 10 ** decimals);
+}
+
+function convertYoctoToNearAndConsiderSmallAmountsToBeZero(indexerRow: IndexerRow): number {
+  let nearAmount = indexerRow.args?.deposit ? indexerRow.args.deposit / YOCTO_CONVERSION_CONSTANT : 0; // converting from yoctonear to near
+  // Round very small transfers down to 0. TODO: Document why this is a good idea or a requirement. Consider improving the name of the constant.
+  nearAmount = Math.abs(nearAmount) >= MINIMUM_AMOUNT ? nearAmount : 0;
+  return nearAmount;
+}
```

Comment:
What was the thought of having these in the main transformations file as oppose to helpers?

Reply:
@nearken Let's figure out together what folders and filenames we want.